fiscalsim_us/tests/policy/baseline/gov/states/mn/tax/income/mn_itemized_deductions.yaml

```diff
@@ -15,7 +15,7 @@
   input:
     filing_status: SINGLE
     adjusted_gross_income: 216_050
-    interest_deduction: 10_000
+    charitable_deduction: 10_000

```

Comment:
@mitchellpound. Why the change from `interest_deduction` to `charitable_deduction`?

Reply:
@rickecon. The `interest_deduction` is not a deduction that is limited over the income threshold, so this test wouldn't test if the code would pick the correct of the two options to phase out the itemized deductions. I changed it to `charitable_deduction` which are limited over the income threshold to make sure the limitation logic was working.